evap/staff/views.py

```diff
@@ -1180,7 +1181,9 @@ def user_import(request):
 def user_edit(request, user_id):
     @receiver(RewardPointGranting.granted_by_removal)
     def notify_reward_points(users, **kwargs):
-        messages.info(request, _('The removal of courses has granted the user "{}" reward points for the active semester.'.format(users[0].username)))
+        messages.info(request, ungettext('The removal of courses has granted the user "{name}" {points} reward point for the active semester.',
+            'The removal of courses has granted the user "{name}" {points} reward points for the active semester.', users[0][1])
+            .format(name=users[0][0].username, points=users[0][1]))
```

Comment:
maybe we shouldn't call the parameter `users` anymore because it's now more than that.
in this case `grant_reward_points_after_delete` could maybe also just return a tuple instead of a one element list with a tuple. @karyon, what do you think?

Reply:
The reason `grant_reward_points_after_delete` fires an event with a list of tuples (which can be a namedtuple for improved readability) is that the removal of participants has got the same way through the code, but could have more affected users. We discussed in the earlier PR the possibiity of two events. Btw, I called `users` while rewriting it `affected` but it had already a collision and I don't want to confuse more than necessary.